### PR TITLE
Doc changes

### PR DESCRIPTION
**Pull request by ahammond**
I threw together some doc updates based on my experience getting `salt-cloud` working tonight. Big thanks to redbeard2 in #salt for all the help!

### CODE REVIEW

File: doc/topics/rackspace.rst [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+the Salt Master.
+You can confirm this by running the following on the salt master.
+
+.. code-block:: bash
+
+    # salt 'foo' test.ping
+
+You should see your new node responding with a True.
+
+Optional Settings
```
Comment (techhat) [inline doc/topics/rackspace.rst:76]: If we don't have any optional settings at the moment, let's leave them out.

File: doc/topics/rackspace.rst [hunk 2]
```diff
@@ -0,0 +1,82 @@
+==============================
+Getting Started With RackSpace
+==============================
+
+RackSpace is the other major public cloud platform and is one of the core
```
Comment (techhat) [inline doc/topics/rackspace.rst]: I wouldn't call it "the" other. I would rather not advertise any one or two cloud platforms above any others.

File: doc/topics/rackspace.rst [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+    # You may need to change the compute_region
+    OPENSTACK.compute_region: DFW
+
+    # You will certainly need to change each of these
+    OPENSTACK.user: myname
+    OPENSTACK.tenant: 123456
+    OPENSTACK.apikey: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+
+Under Hosting -> Cloud Servers, I see that my servers are provisioned in
```
Comment (techhat) [inline doc/topics/rackspace.rst]: First person should be avoided in technical documentation at all costs.

File: doc/topics/rackspace.rst [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+Set up an initial profile at ``/etc/salt/cloud.profiles``:
+
+.. code-block:: yaml
+
+    openstack_512:
+        provider: openstack
+        size: 512MB Standard Instance
+        image: Ubuntu 12.04 LTS (Precise Pangolin)
+
+You can now instantiate a machine based on this profile with a salt
```
Comment (techhat) [inline doc/topics/rackspace.rst]: Second person should also be avoided. Maybe something like, "A machine based on this profile may now be instantiated with a salt command".

File: doc/topics/rackspace.rst [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+command:
+
+.. code-block:: bash
+
+    # salt-cloud -p openstack_512 foo
+
+This will create a virtual machine at RackSpace with the name ``foo``.
+It took about 5 minutes to complete when I tried it.
```
Comment (techhat) [inline doc/topics/rackspace.rst]: Leave out exact times, so as not to raise expectations. I have seen this take anywhere from 2 to 15 minutes on Rackspace, and everyone's mileage will vary.

**Comment by techhat** [comment]
Don't let all of my inline comments discourage you. This is good stuff that is sorely needed, but it does need a little work before it can be accepted.

**Comment by ghost** [comment]
agreed @ahammond ! Please commit some updates per notes above & those commits will automatically appear here in the pull req.

File: doc/topics/rackspace.rst (hunk 1 above)
Comment (ahammond) [inline doc/topics/rackspace.rst:76]: I think that positively saying that there are no optional settings is better. I'll update it that way.

File: doc/topics/rackspace.rst (hunk 5 above)
Comment (ahammond) [inline doc/topics/rackspace.rst]: Reworded to clarify and avoid setting expectations.

**Comment by techhat** [comment]
It's better, but there's still some first person in there. But I'm seeing some other first person as well, so I'm going to merge this, and then see if I can do some overall cleanup. Thanks for your help, @ahammond!

**Pull request merged**